package.json

```diff
@@ -63,7 +65,9 @@
     "istanbul": "^0.4.5",
     "ls-engines": "^0.7.0",
     "markdown-magic": "^2.6.0",
+    "markdownlint-cli": "^0.8.0 || ^0.32.2",
```

Comment:
[0.8.0](https://github.com/igorshubovych/markdownlint-cli/blob/d4126d7d6d05b3f2eb5df282404108fc7859ddfc/package.json#L10) is the final version that still supported Node 4 and we need to ensure this is still installable on Node 4 since we test that on CI.

Reply:
Don't we need to manually ensure that in the CI config?

Reply:
I was hoping that npm would automatically choose the compatible version from this list based on the current Node version? If CI fails, I'll look into updating the CI config.

Reply:
npm 7+ definitely does that, but older npms (which are needed on older nodes) don't, afaik. totally fine to wait and see what CI does tho

Reply:
It looks like CI passes already so I'll leave as-is.